barretenberg/cpp/pil/vm2/execution.pil

```diff
@@ -92,23 +62,180 @@ sel * (1 - sel') * (1 - last) = 0;
 #[LAST_IS_LAST]
 last * sel' = 0;
 
-// These are needed to have a non-empty set of columns for each type.
-pol public input;
+/**************************************************************************************************
+ *  Temporality group 1: Instruction fetching (unconditional)
+ **************************************************************************************************/
 
-#[EXEC_SPEC_READ]
+pol commit bytecode_id;
+pol commit instr_length; // Need this for shift col on next_pc
+pol commit sel_instruction_fetching_failure;
+
+pol commit ex_opcode;
+pol commit indirect;
+pol commit op[6];  // operands
```

Comment:
How many operands do we actually need? I mistakenly changed it to 6 here. Can do 7, but I can't be bothered too much until we actually need all of them.